guides/05_chatbots/04_the-retry-and-undo-events.md

```diff
@@ -0,0 +1,115 @@
+# Retrying and Undoing Messages
```

Comment:
Suggestion: I would rename this Guide something like: "Chatbot Events: Retrying, Undoing, and Liking/Disliking Messages"

and move the section "Liking / Disliking Chat Messages" from the `creating-a-custom-chatbot-with-blocks` guide here. It makes sense to include the `.like` event in the same guide. If you want, you could keep the `.like` section in the original guide, but then also repeat it here, because I think its nice to have them all in one place for easy reference (and you could include a link to this Guide from that one)

Reply:
Good idea - I moved the like section from the blocks guide to this new guide